src/OrchardCore.Modules/OrchardCore.Email/Services/SmtpSettingsConfiguration.cs

```diff
@@ -28,6 +28,9 @@ public void Configure(SmtpSettings options)
                 .GetAwaiter().GetResult()
                 .As<SmtpSettings>();
 
+            if (settings == null)
```

Comment:
Not necessary anymore, I pulled @paynecrl97 's PR

Reply:
If you care if the settings have been set or not, you can use `Has<>()`